README.md

```diff
@@ -1,15 +1,19 @@
 # baytrailbirding
-
-[![Build Status](https://travis-ci.com/rchunter/baytrailbirding.svg?branch=main)](https://travis-ci.com/rchunter/baytrailbirding)
+![Build Status](https://github.com/debbieyuen/baytrailbirding/actions/workflows/ci.yml/badge.svg)
```

Comment:
This seems to link to the SVG file? Shouldnt it link somewhere else?

Reply:
I'm not sure where this should link to. I've looked at other teams badges and they seem to link also link to the SVG file

Reply:
Okay, yeah, seems like there isn't a standard place to link to, carry on!